### PR TITLE
EmailApi with query and params use 

### PR DESCRIPTION
**Pull request by jitendrasbunde**

### CODE REVIEW

File: emailApi/email.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+exports.mailSend = (mailObj)=>{
+	const nodemailer = require('nodemailer');
+	const transport = nodemailer.createTransport({
+		host: 'smtp.gmail.com',
+		port: 465,
+		service  : 'gmail',
+		auth :{
+			user :'jitendrasbunde@gmail.com',
+			pass :'gawgialpaiqhongo'
```
Comment (sunil4sonawane) [inline emailApi/email.js:10]: remove password

File: emailApi/emailApi.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+		to :req.query.to+","+req.query.cc,
+		subject  :'Email API test',
+	 	text :'Node Mail API Testing mail By Jitendra'
+  }
+  email.mailSend(obj);
+   res.json({'url':'post  call in emailApi.js'});
+}
+);
+
+module.exports = router;
```
Comment (sunil4sonawane) [inline emailApi/emailApi.js:33]: add new line

File: emailApi/index.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const email = require('./emailApi')
+const app = express();
+
+app.use('/email',email);
+app.listen(3007,function(){
+	console.log('Server is running on Port 3007');
+}
+);
```
Comment (sunil4sonawane) [inline emailApi/index.js:9]: add new line

**Review by sunil4sonawane — CHANGES_REQUESTED** [review]
Check comments

File: emailApi/email.js (hunk 1 above)
Comment (jitendrasbunde) [reply emailApi/email.js:10]: means sir i don't get it

**Review by sunil4sonawane — CHANGES_REQUESTED** [review]
You have raised this against master. please change it

**Pull request closed without merging**